src/thirdparty/uuid.js

```diff
@@ -0,0 +1,247 @@
+//     uuid.js
```

Comment:
Is there a minified version we could use here instead or will it automatically be minified when a release is build?